### ISSUE TITLE
[Idea] create a awesome-neuromore repo where people could share examples of classifiers/models and state machines

### ISSUE THREAD

**Issue by srghma** [neuromore/studio#124]
as a sidequestion: 
1. where are the examples stored (in github) that are being distributed with neuromore studio .exe? could someone point me to the directory in github
2. does someone know if someone already shares such examples of classifiers? (that could be included in awesome-neuromore repo)

I'm looking for alpha-theta training protocol example

**Comment by srghma** [comment]
everything is stored online, that's a bummer

**Comment by stellarpower** [comment]
I'd like to see this too, a number of examples are really trivial, that display geatures available, and then there is a full classifier that is so large and complicated it's very hard to follow what is going on at a high-level with no ability to collapse groups of nodes in.
